pbrd/pbr_zebra.c

```diff
@@ -549,8 +549,8 @@ static void pbr_encode_pbr_map_sequence(struct stream *s,
 	stream_put(s, ifp->name, INTERFACE_NAMSIZ);
 }
 
-void pbr_send_pbr_map(struct pbr_map_sequence *pbrms,
-		      struct pbr_map_interface *pmi, bool install, bool changed)
+int pbr_send_pbr_map(struct pbr_map_sequence *pbrms,
```

Comment:
why is a int better than a boolean here?  I am abit confused about what we are supposed to be returning?  true that we did not send to zebra?

Reply:
sure i can make it a bool